config.json

```diff
@@ -1801,8 +1801,8 @@
       {
         "groupId": "com.squareup.okhttp3",
         "artifactId": "logging-interceptor",
-        "version": "4.5.0",
-        "nugetVersion": "4.5.0",
+        "version": "4.6.0",
```

Comment:
This is not just a patch bump, this is a new Java version.

Reply:
I had to manually bump some (remember order of last updates FB was 1st but we merged it as last).
Will fix right now.

Reply:
fixed 4.6.0.1